lib/utils/form-builder/groupsHistory.ts

```diff
@@ -32,3 +33,51 @@ export const getPreviousIdFromCurrentId = (currentId: string, history: string[])
   }
   return null;
 };
+
+export const getRelevantValues = (
```

Comment:
Let's add a doc block here to explain what this function is doing.

Reply:
i.e. not sure what "Relevant" means

Reply:
What about filterValuesByVisitedGroup()